scripts/build-charts-ci.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+
+# Install chartmuseum using the gofish package manager
+curl -fsSL https://raw.githubusercontent.com/fishworks/gofish/main/scripts/install.sh | bash
+gofish init
+gofish install chartmuseum
+
+# We need the helm push plugin to automatically package and push chart to our repo
+helm plugin install https://github.com/chartmuseum/helm-push.git
+
+# Systemd unit file to start chartmuseum in the runner
+cat <<EOF | sudo tee /etc/systemd/system/chartmuseum.service
+[Unit]
+Description=Helm Chartmuseum
+Documentation=https://chartmuseum.com/
+
+[Service]
+ExecStart=/usr/local/bin/chartmuseum \\
+ --debug \\
+ --port=8090 \\
+ --storage="local" \\
+ --storage-local-rootdir="/home/${USER}/chartstorage/"
+Restart=on-failure
+RestartSec=5
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl start chartmuseum
+
+# Waiting chartmuseum initialization
+sleep 5
+
+# Add our new ephemeral repo
+helm repo add chartmuseum http://localhost:8090
+helm repo update
+
+# Push all our charts to the repo
+# We force version 0.1.0 because we don't care about versionning
+# We only care about testing here
+cd chart/
+helm cm-push --version "0.1.0" epinio-installer/ chartmuseum
```

Comment:
Shouldn't there be some change in this PR to install from this location? (maybe it's already there and I can't see it?)

Reply:
We install from this location thank to https://github.com/epinio/helm-charts/pull/52/files#diff-4b4a9fbed581194852da125a443f567a145dbcadd7a0ea64be79f50f8470ec51R51
Am I wrong?

Reply:
Yes we don't need to push epinio-installer to chartmuseum as chart testing is building it from dir. Let me remove it.